vm/systemSmartContracts/staking.go

```diff
@@ -1427,6 +1427,11 @@ func (r *stakingSC) EpochConfirmed(epoch uint32) {
 	log.Debug("stakingSC: stake/unstake/unbond", "enabled", r.flagStake.IsSet())
 }
 
+// SetNewGasCosts is called whenever a gas cost was changed
+func (r *stakingSC) SetNewGasCosts(gasCost vm.GasCost) {
+	r.gasCost = gasCost
```

Comment:
We need to protect `s.gasCost` with a mutex

Reply:
no need. there is no possibility to call the from different go routines

Reply:
protected all the way